src/icons/QuickFilterIcon.js

```diff
@@ -0,0 +1,9 @@
+import { BaseSvg } from 'icons';
+
+export default ({ fill = '#fff', width = 14, height = 14, ...props }) =>
+  BaseSvg({
+    svg: `<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 511.99 512.01"><defs><style>.cls-1{fill:${fill};}</style></defs><title>icon-quick-filters</title><path class="cls-1" d="M488,0H24C2.71,0-8,25.87,7.06,41L192,225.94V432a24,24,0,0,0,10.24,19.66l80,56C298,518.69,320,507.49,320,488V226L505,41C520,25.9,509.34,0,488,0Z" transform="translate(-0.02)"/></svg>`,
+    width: `${width}px`,
+    height: `${width}px`,
+    ...props,
+  });
```

Comment:
I think some of these icons already exist in the portal, at least the following:
- quick filters
- study
- demographic
- biospecimens

I could be missing some, but I'd suggest trying out some of the existing icons and coloring them first. Wanna be a bit more mindful of the bundle size :P

Reply:
Great! -  I'll double check these

Reply:
![screen shot 2019-01-30 at 2 04 12 pm](https://user-images.githubusercontent.com/11821106/52006658-ac8d0f80-249a-11e9-9ac0-eafd7bfb6869.png)
![screen shot 2019-01-30 at 2 03 31 pm](https://user-images.githubusercontent.com/11821106/52006659-ac8d0f80-249a-11e9-91c2-03090ec02bc6.png)

Reply:
existing icons are currently in assets, so we'll leave reusability for later to avoid refactoring :) thanks @ciaranschutte for looking into it!